code/__DEFINES/factions.dm

```diff
@@ -1,5 +1,6 @@
 #define FACTION_NEUTRAL "Neutral"
 #define FACTION_TERRAGOV "TerraGov"
+#define FACTION_RETIRED "TerraGov Retiree Force"
```

Comment:
Just a note that being a different faction means they can't give/receive orders to tgmc faction people.